homework3/semi-uber/Server/api/Schemas/user.schema.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require("mongoose");
+const uniqueValidator = require("mongoose-unique-validator");
+const Schema = mongoose.Schema;
+const User = new Schema({
+  name: { type: String, require: true, minLength: [3, "Name is too short!"] },
```

Comment:
and what about Li, Po ? :) They will not be able to work in your system?)

Reply:
Fixed